bundler/lib/dependabot/bundler/native_helpers.rb

```diff
@@ -36,7 +35,7 @@ def self.versioned_helper_path(bundler_version:)
       end
 
       def self.helper_path(bundler_version:)
-        "ruby #{File.join(versioned_helper_path(bundler_version: bundler_version), 'run.rb')}"
+        "bundle exec ruby #{File.join(versioned_helper_path(bundler_version: bundler_version), 'run.rb')}"
```

Comment:
This is needed to force the `BUNDLER_VERSION` env to take effect for bundler v1 and force bundler to pick the lastest v2 `2.2.20` instead of always falling back to the pre-installed version in ruby 2.7 (`2.1.4`)

Reply:
Seems like a good change to run this in the bundle environment regardless 👍